oembed/v1.0/oembed.md

```diff
@@ -0,0 +1,64 @@
+# ClickView oEmbed API
+
+[oEmbed](https://oembed.com/) is an open format designed to allow embedding content from a website into another page.
+
+The ClickView oEmbed API can be used to obtain the the necessary information to embed ClickView videos into your website.
+
+
+## Resource URL
+
+```http
+GET https://online.clickview.com.au/oembed
+GET https://online.clickview.co.uk/oembed
+GET https://online.clickview.co.nz/oembed
+```
+
+## Parameters
+
+| Name | Required | Description | Default | Example |
+| ---- | -------- | ----------- | ------- | ------- |
+| url | required | The URL of the resource you are trying to embed || `https://clickv.ie/w/{short-code}` |
+| format | optional | The response format. Supported values are `json` and `xml`. | `json` | `json` |
+
+## Supported URLs
+oEmbed is supported on the following pages:
+
+**Shared Links**:
+* `https://clickv.ie/w/{short-code}`
+
+**Videos**:
+
+* `share?sharecode={share-code}`
+* `share/embed?sharecode={share-code}`
+* `exchange/video/{video-id}/*`
+* `exchange/videos/{video-id}/*`
+* `exchange/topics/{topic-id}/*/videos/{video-id}/*`
+* `exchange/categories/{category-id}/*/videos/{video-id}/*`
+* `exchange/series/{series-id}/*/videos/{video-id}/*`
+* `exchange/channels/{channel-id}/*/videos/{video-id}/*`
+
+**Playlists**:
+
+* `playlists/share/{share-code}/*`
+* `playlists/{playlist-id}/videos/{video-id}/*`
+* `exchange/channels/{channelId}/*/playlists/*/`
```

Comment:
We don't support playlist links unless they're a user channel playlist. At the moment OEmbedController returns nothing for non user channel playlists

Reply:
Also you're using kebab-case everywhere else but here you are using lower camel case. Since we use lower camel in the image API docs I would say that should be how we define url parameters. (see https://github.com/clickviewapp/Docs/blob/master/image/v2.0/images.md)